### PR TITLE
Feature/asset holdings

### PR DESCRIPTION
**Pull request by SNuseibeh**

### CODE REVIEW

File: src/utils/constants/assetTimeseries.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+];
+
+export const assetGraphView = [
+  {
+    title: "Price",
+    value: "price",
+  },
+  {
+    title: "Trading View",
+    value: "tradingview",
```
Comment (almujtabamohammed) [inline src/utils/constants/assetTimeseries.ts:53]: Translate

File: src/components/shared/tables/exchangeHoldingTable.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+              <tr key={index} className="hover:bg-grey-1 cursor-pointer h-16">
+                <td className="font-medium leading-6 text-white pl-5">
+                  {CapitalizeString(elm.provider_name.toLowerCase())}
+                </td>
+                <td className="font-medium leading-6 text-white">
+                  {formatNumber(elm.free)} {asset.symbol.toUpperCase()}
+                </td>
+                <td className="font-medium leading-6 text-white">
+                  {formatNumber(elm.avg_price, true)}
+                </td>
```
Comment (almujtabamohammed) [inline src/components/shared/tables/exchangeHoldingTable.tsx:61]: elm.avg_price is always 0?

File: src/components/shared/tables/assetOrders.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+export const AssetOrdersTable: FC = () => {
+  const { t } = useTranslation(["asset"]);
+  const orders = useSelector(getOrders);
+  const asset = useSelector(getAsset);
+
+  const header = [
+    t("type"),
+    t("status"),
+    t("amount"),
+    t("amount"),
```
Comment (almujtabamohammed) [inline src/components/shared/tables/assetOrders.tsx:20]: Replace the 2nd "amount" with price

**Review by almujtabamohammed — CHANGES_REQUESTED** [review]
Assets page isn't responsive

File: src/utils/constants/assetTimeseries.ts (hunk 1 above)
Comment (SNuseibeh) [reply src/utils/constants/assetTimeseries.ts:53]: Not used in the app, removed from codebase

File: src/components/shared/containers/asset/assetVote.tsx [hunk 4]
```diff
@@ -5,13 +5,13 @@ import {
...
+  const { t } = useTranslation(["asset"]);
   return (
     <Col className="gap-3">
-      <p className="font-medium text-sm">
-        How do you feel about Bitcoin today?
-      </p>
+      <p className="font-medium text-sm">{t("assetvoting")}</p>
       <Row className="gap-2">
         <ShadowButton
           title="Bullish"
```
[the remark this answers is not in the document]
Comment (SNuseibeh) [reply src/components/shared/containers/asset/assetVote.tsx:17]: this is handled with the voting PR

**Pull request merged**